tests/test_serializers.py

```diff
@@ -15,6 +15,9 @@
 drf2 = rest_framework.VERSION < StrictVersion('3.0.0')
 drf3 = rest_framework.VERSION >= StrictVersion('3.0.0')
 
+django_strict_version = StrictVersion('%s.%s.%s' % django.VERSION[:3])
+django2 = django_strict_version > StrictVersion('1.10.0')
```

Comment:
And then Django 1.11 happened ;-)

In any case, the older PR https://github.com/GetBlimp/django-rest-framework-jwt/pull/398 did this.